packages/ListBox/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import Button from "@paprika/button";
+import PropTypes from "prop-types";
+import { FooterContainerStyled } from "./Footer.styles";
+import useListBox from "../../useListBox";
+import applyOnChange from "../../helpers/applyOnChange";
+
+const propTypes = {
+  isAcceptVisible: PropTypes.bool,
+  isCancelVisible: PropTypes.bool,
+  isClearVisible: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  kindAccept: PropTypes.string,
+  kindCancel: PropTypes.string,
+  kindClear: PropTypes.string,
+  labelAccept: PropTypes.string,
+  labelCancel: PropTypes.string,
+  labelClear: PropTypes.string,
+  onClickAccept: PropTypes.func,
+  onClickCancel: PropTypes.func,
+  onClickClear: PropTypes.func,
+  size: PropTypes.string,
+};
+
+const defaultProps = {
+  isAcceptVisible: true,
+  isCancelVisible: true,
+  isClearVisible: true,
+  isDisabled: false,
+  kindAccept: "primary",
+  kindCancel: "minor",
+  kindClear: "minor",
+  labelAccept: "Accept",
+  labelCancel: "Cancel",
+  labelClear: "Clear",
+  onClickAccept: null,
+  onClickCancel: null,
+  onClickClear: null,
+  size: "small",
+};
+
+export function FooterComponent(props, ref) {
+  const [, dispatch] = useListBox();
+
+  const {
+    labelCancel,
+    labelAccept,
+    labelClear,
+    isCancelVisible,
+    isAcceptVisible,
+    isClearVisible,
+    kindAccept,
+    kindCancel,
+    kindClear,
+    size,
+  } = props;
+
+  const handleClickAccept = event => {
+    event.stopPropagation();
+
+    dispatch({
+      type: useListBox.types.accept,
+      payload: { onChangeFn: applyOnChange(props.onClickAccept, "listbox:footer:accept") },
```

Comment:
I'm not yet sure what these string literals are needed for... but shouldn't they be defined as consts or enums somewhere?

Reply:
these are the kind of triggered event, is just notifying the consumer, nothing fancy is happening with them we can remove them later if we feel like it.